frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Override/hook/useCustomConfigType.ts

```diff
@@ -0,0 +1,15 @@
+import { useTranslation } from 'react-i18next';
+
+type CustomConfigType = {
```

Comment:
Great that you have created a hook for this! 👏 
Based on my comments above I'll suggest exporting this, or maybe moving it to a `types` folder 😄